docs/articles/baseplatform/how-to-change-bootsplash.rst

```diff
@@ -0,0 +1,50 @@
+:orphan:
+
+Changing bootsplash
+===================
+
+PELUX uses psplash for graphical boot splash screen. PELUX comes with reference images
+come with a default boot splash screen containing a dark background and the PELUX logo.
```

Comment:
this sentence looks weird, `comes with ... come with`

Is there a need to describe that it's a black background? Perhaps it's enough to state that it's a boot splash screen with a PELUX logo, I mean if we change the logo we need to update this text too.